docs/release-notes/index.md

```diff
@@ -34,6 +34,7 @@ To check for security updates, go to [Security announcements for the Elastic sta
 
 - Add checks for the Resty response object in all Meraki module API calls to ensure proper handling of nil responses. [44193]({{beats-pull}}44193)
 - Add a latency configuration option to the Azure Monitor module. [44366]({{beats-pull}}44366)
+- Add GCP cache for metadata collectors. [44432]({{beats-pull}}44432)
```

Comment:
Did this change go into 9.0.2 or is it going in a future release? This line is in the 9.0.2 section of the release notes. If the fix didn't make it into 9.0.2, you should add it to the appropriate 9.0.x section instead (more towards the top of this file).

Reply:
it's going in `9.1.0` actually, I don't see that version present in the file where should it be added?

Reply:
Yes exactly!